diff --git a/CMakeLists.txt b/CMakeLists.txt
index fb23b7d9..97373b92 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -142,7 +142,7 @@ test_big_endian(CPU_IS_BIG_ENDIAN)
 check_c_compiler_flag(-Werror HAVE_WERROR_FLAG)
 check_c_compiler_flag(-Wdeclaration-after-statement HAVE_DECL_AFTER_STMT_FLAG)
 check_c_compiler_flag(-mstackrealign HAVE_STACKREALIGN_FLAG)
-check_cxx_compiler_flag(-Weffc++ HAVE_WEFFCXX_FLAG)
+#check_cxx_compiler_flag(-Weffc++ HAVE_WEFFCXX_FLAG)
 
 if(MINGW AND (WITH_FORTIFY_SOURCE OR WITH_STACK_PROTECTOR))
   check_library_exists("ssp.a"  __stack_chk_fail "" HAVE_LIBSSP)
@@ -169,7 +169,6 @@ add_compile_options(
     $<$<BOOL:${MSVC}>:/wd4267>
     $<$<BOOL:${MSVC}>:/wd4996>
     $<$<BOOL:${ENABLE_WERROR}>:-Werror>
-    $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<BOOL:${HAVE_WEFFCXX_FLAG}>>:-Weffc++>
     $<$<AND:$<COMPILE_LANGUAGE:C>,$<BOOL:${HAVE_DECL_AFTER_STMT_FLAG}>>:-Wdeclaration-after-statement>)
 
 if(WITH_FORTIFY_SOURCE AND HAVE_LIBSSP)
