diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12a2416d..97ad20a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -188,33 +188,10 @@ set(LIB_VERSION_NUM ${LIB_MAJOR_VERSION},${LIB_MINOR_VERSION},${LIB_REVISION},0)
 
 set(EXPORT_DECL "")
 
-
-# Some systems erroneously require the __STDC_FORMAT_MACROS macro to be defined
-# to get the fixed-width integer type formatter macros.
-check_cxx_source_compiles("#include <cinttypes>
-#include <cstdio>
-int main()
-{
-    int64_t i64{};
-    std::printf(\"%\" PRId64, i64);
-}"
-HAVE_STDC_FORMAT_MACROS)
-if(NOT HAVE_STDC_FORMAT_MACROS)
-    set(CPP_DEFS ${CPP_DEFS} __STDC_FORMAT_MACROS)
-endif()
-
 # Some systems may need libatomic for atomic functions to work
 set(OLD_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES})
 set(CMAKE_REQUIRED_LIBRARIES ${OLD_REQUIRED_LIBRARIES} atomic)
-check_cxx_source_compiles("#include <atomic>
-std::atomic<int> foo{0};
-int main() { return foo.fetch_add(2); }"
-HAVE_LIBATOMIC)
-if(NOT HAVE_LIBATOMIC)
-    set(CMAKE_REQUIRED_LIBRARIES "${OLD_REQUIRED_LIBRARIES}")
-else()
-    set(EXTRA_LIBS atomic ${EXTRA_LIBS})
-endif()
+set(EXTRA_LIBS atomic ${EXTRA_LIBS})
 unset(OLD_REQUIRED_LIBRARIES)
 
 if(ANDROID)
