diff --git a/src/cpp/piper.hpp b/src/cpp/piper.hpp
index 21f0ece..332a619 100644
--- a/src/cpp/piper.hpp
+++ b/src/cpp/piper.hpp
@@ -9,9 +9,9 @@
 #include <vector>
 
 #include <onnxruntime_cxx_api.h>
-#include <piper-phonemize/phoneme_ids.hpp>
-#include <piper-phonemize/phonemize.hpp>
-#include <piper-phonemize/tashkeel.hpp>
+#include <phoneme_ids.hpp>
+#include <phonemize.hpp>
+#include <tashkeel.hpp>
 
 #include "json.hpp"
 
